src/appleseed-max-impl/utilities.cpp

```diff
@@ -742,18 +743,12 @@ namespace
             return new MaxProceduralTextureSource(m_texmap, m_time);
         }
 
-        asf::Tile* load_tile(
+        asr::TilePtr load_tile(
             const size_t                tile_x,
             const size_t                tile_y) override
         {
-            return nullptr;
-        }
-
-        void unload_tile(
-            const size_t                tile_x,
-            const size_t                tile_y,
-            const asf::Tile*            tile) override
-        {
+            asf::Tile* tile = nullptr;
+            return asr::TilePtr::make_non_owning(tile);
```

Comment:
I'll also add a `TilePtr::make_nullptr()` static method to make this cleaner.

Reply:
Great, I was not sure if my implementation was the correct way to do it.

Reply:
Your implementation is perfectly correct.